examples/cpp/core/bf/main.cpp

```diff
@@ -113,7 +113,7 @@ void TranscodeFromTo(Reader& reader, Writer& writer, const Options& options)
 {
     if (!options.schema.empty())
     {
-        bond::SchemaDef schema(LoadSchema(options.schema));
+        bond::SchemaDef schema(LoadSchema(options.schema.front()));
```

Comment:
This doesn't look like it handles an empty string like the old code used to.
Based on my reading and testing of [`boost::escaped_list_separator`][els], which is used for the cmd arg parsing, it will return an empty string for the first item in an input like  `--schema=,second.schema`.


[els]: http://www.boost.org/doc/libs/1_63_0/libs/tokenizer/escaped_list_separator.htm